DataConnectors/ProofpointPOD/ProofpointPOD_API_FunctionApp.json

```diff
@@ -0,0 +1,144 @@
+{
+    "id": "ProofpointPOD",
+    "title": "Proofpoint On Demand Email Security (Preview)",
```

Comment:
Please remove (Preview) from title. It will be added automatically when connector is in preview.

Reply:
Done